audb/core/info.py

```diff
@@ -417,6 +490,10 @@ def splits(
     Returns:
         splits of database
 
+    Example:
+        >>> splits('emodb', version='1.1.0')
```

Comment:
Mhh, it's not nice that `emodb` does not define `splits`. But I guess it's not worth switchting to another database just because of this example,

Reply:
We don't have another database yet in the public repository.
We could consider using another one, after we have published more, but I guess this will take some time.